site/pages/reference/account-kit/core/functions/createConfig.mdx

```diff
@@ -27,7 +27,7 @@ import { sepolia } from "@account-kit/infra";
 
 const config = createConfig({
   chain: sepolia,
-  apiKey: "your-api-key",
```

Comment:
the basic usage of this for a devs doesn't change much, just requires passing in a `transport` for a given chain.

For multi chain, that's also been made better where you can provide one global `transport` or you can pass a `chains` array which contains `transports` for each chain